CONTRIBUTING.rst

```diff
@@ -0,0 +1,104 @@
+.. _contributing:
```

Comment:
What is this label needed for?

Isn't it enough to link to the whole file?

Reply:
Removed it.